CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Foodcritic Changelog:
 
+## [11.2.0](https://github.com/acrmp/foodcritic/tree/v11.2.0) (...)
+
+[Full Changelog](https://github.com/acrmp/foodcritic/compare/v11.1.0...v11.2.0)
+
+**Implemented enhancements:**
+
+- Disabled `FC078` by default to simply use of Foodcritic for non-community cookbook developers. To enable it again: `foodcritic -t any .`
```

Comment:
We should not that we disabled the opensource tag which includes FC078 by default. There will probably be more things tagged with opensource in the future and this makes it sound like we made the default ~FC078